test-cases/raft/raft-topology-changes-double-cluster-size-12h.yaml

```diff
@@ -0,0 +1,26 @@
+test_duration: 800
```

Comment:
same + the name of this test isn't clear.
What does it mean changes-double-cluster-size?
Pick a better name.